ADD_NEW_DATASET.md

```diff
@@ -22,7 +22,7 @@ This creates a copy of the code under your GitHub user account.
 
 Now you are ready, each time you want to add a new dataset, follow the steps in the following section:
 

```

Comment:
looks like your PR has changes in other files than `isizulu_ner_corpus`, could you please remove those ?

Also if it's a rebase/merge issue, feel free to create another branch and a new clean PR

Reply:
Seen the two languages have been added with the branch.  I assume no more actions is needed here

Reply:
Indeed ! thanks again for adding both of them.

Closing this PR then :)